soh/src/code/z_en_item00.c

```diff
@@ -520,6 +520,7 @@ void EnItem00_Init(Actor* thisx, GlobalContext* globalCtx) {
                 func_8002F554(&this->actor, globalCtx, getItemId);
             } else {
                 getItem = Randomizer_GetItemFromActor(this->actor.id, globalCtx->sceneNum, this->ogParams, getItemId);
+                getItem.getItemFrom = ITEM_FROM_FREESTANDING;
```

Comment:
since there's logic here for where the item is from, would it maybe make sense to have the default `getItemFrom` value be `ITEM_FROM_UNKNOWN` or something?

Reply:
that would require actor file changes as discussed above, keeping the default as `ITEM_FROM_NPC` works